target/scripts/helpers/log.sh

```diff
@@ -42,32 +42,22 @@ RESET=$(echo -ne   '\e[0m')
 # If the first argument is not set or invalid, an error
 # message is logged. Likewise when the second argument
 # is missing. Both failures will return with exit code '1'.
-function _log
-{
+function _log() {
```

Comment:
I saw your [associated commit message](https://github.com/docker-mailserver/docker-mailserver/pull/2752/commits/d5e33f5f9ac36325a2f43d66f635ab8da0486adf) for this, but I'm not sure what you're referencing?

We generally keep to that style now don't we? Has there been complaints somewhere? I think the only case that doesn't happen is with `.bats`, or in my case if I forget to style it that way and it slips through review (_in languages I'm used to, it's common to take this approach with `{` on the same line_).

Personally, unless there's a good reason to, we should already be pretty consistent with the function style used, I don't think we need to slowly lose that consistency (_or do a big PR to change it_). If another maintainer wants that, I'm fine with it, but it is a gripe I have when trawling history via `git blame` for more information on something to wade through these kind of broad sweeping adjustments.

Reply:
Agreed. I always thought people hate me for doing this. I am used to writing it like this in C and Rust.. and I am happy to uphold this style :D 😆

Reply:
> I always thought people hate me for doing this.

I do 😆 It's just that I don't like wasting a line for no benefits. The some goes for if statements, I prefer for simplicity:

```bash
if true; then
    foo
fi
```

So if there are plans to change that, I am all for it 👍 However I understand that I am not the only maintainer, so my personal style wishes may not always meet those of the others. Most important for me is consistency across our code base, even if I don't like the decision that was chosen 😆

Reply:
@casperklein I have no problem with a PR that re-styled _the whole_ code base at once 😆 But I guess this also has not benefit. I would just sick to the style I imposed upon all of you 😆 😆

Reply:
> Agreed. I always thought people hate me for doing this.

Oh, totally not a fan, but since you've already converted the code-base I just go with it. Not a big deal.

> Most important for me is consistency across our code base, even if I don't like the decision that was chosen :laughing: 

This.

---

If someone wants to do that mundane task of switching a style through all the files, go for it... Just wait until after a release. I especially don't want to have to rebase my WIP PR against it :sweat_smile: